### DISCUSSION TITLE
Webpage with a location of all the GS

### DISCUSSION THREAD

**Discussion by lu9cbl** [dernasherbrezon/r2cloud#117] [Q&A] [answered]
Hi to all! i'm installing one ground station, and i want to know if its there any map, to the location of all groundstations??? similar to the map that has Satnogs, for example.

Thats help to see if any neighbord has a station in a near location.
Thanks for your effort.

**Comment by dernasherbrezon** [comment, marked as answer]
Done.
<img width="1438" alt="Screenshot 2022-01-31 at 22 17 06" src="https://user-images.githubusercontent.com/1614424/151883024-4ee597f6-fd3a-482d-8f73-c869d82d6451.png">
